layouts/index.js

```diff
@@ -0,0 +1,35 @@
+import Background from "../components/Background";
+import Footer from "../components/Footer";
+import Header from "../components/Header";
+import PropTypes from "prop-types";
+
+const Layout = ({
+  children,
+  hideBackground,
+  hideFooterBanner,
+  footerCompactView,
+}) => {
+  return (
+    <div className="page min-h-screen flex flex-col justify-start justify-items-stretch overflow-hidden pt-[60px] laptop:pt-[110px] bg-purple-dark relative">
```

Comment:
This is not working as it used to. On the site right now, this provides padding-top of either 65px or 110px; in your PR that padding is missing.

Reply:
Yes, we are using extra padding top in `index.js`. For index.js it is `143px` and for other pages it is `110px`.
So we have to pass the padding top value to the Layout component.